src/index.html

```diff
@@ -1,15 +1,118 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          onsubmit="onSubmit()"
+          autocomplete="off">
+      <fieldset class="feildset">
+        <legend>Personal information</legend>
+        <label class="label" for="surname">Surname:</label>
+        <input type="text" id="surname"
+               name="surname" class="bottom"
+               required>
```

Comment:
If the element has three or more attributes - they all should start on next line.

```suggestion
        <input 
          type="text"
          id="surname"
          name="surname"
          class="bottom"
          required
        >
```